mayastor/src/core/handle.rs

```diff
@@ -193,6 +196,63 @@ impl BdevHandle {
             Err(CoreError::ResetFailed {})
         }
     }
+
```

Comment:
please comment all the functions added here.

Reply:
Fixed.